src/types/xdai/xdai.ts

```diff
@@ -82,6 +49,23 @@ export class Xdai extends Ethereum {
           },
         ];
         break;
+      case NodeClient.NETHERMIND:
+        versions = [
+          {
+            version: '1.12.3',
+            clientVersion: '1.12.3',
+            image: 'nethermind/nethermind:1.12.3',
+            dataDir: '/nethermind/nethermind_db',
+            walletDir: '/nethermind/keystore',
+            configPath: '/nethermind/configs/xdai.cfg',
+            networks: [NetworkType.MAINNET],
+            breaking: false,
+            generateRuntimeArgs(data: CryptoNodeData): string {
+              return ` --config xdai`;
```

Comment:
Is `xdai` the regular name of the config file for xdai using nethemind? We try and stick with the project conventions as much as possible, so for most geth implementations we use `config.toml` for example. Should this be `xdai.json`? Or, is the most common config name something else? If it's just xdai, that's fine. But, I want to match conventions so that anyone using node launcher is immediately familiar with what is what.

Reply:
Ok, noted.  The file is in `/nethermind/configs/xdai.cfg` and that form for the --config param came from their docs:
https://docs.nethermind.io/nethermind/ethereum-client/docker#available-configurations

Reply:
I see that it matches their recommendation. Perfect.